js/client/modules/@arangodb/testsuites/resilience.js

```diff
@@ -93,6 +93,7 @@ function activeFailover (options) {
   let testCases = tu.scanTestPaths(testPaths.active_failover);
   options.activefailover = true;
   options.singles = 4;
+  options.disableMonitor = true;
```

Comment:
please clone the options object, else it will override the settings for subsequent other testsuites.